Specs/Sequence/Swedish/PhoneNumberModel.json

```diff
@@ -0,0 +1,210 @@
+[
+  {
+    "Input": "Tel: 08-123 456 67",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "08-123 456 67",
+        "Start": 5,
+        "End": 17,
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "score": "1",
+          "value": "08-123 456 67"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Tel: 08-123 45 67",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "08-123 45 67",
+        "Start": 5,
+        "End": 16,
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "score": "1",
+          "value": "08-123 45 67"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Tel: 08-12 45 67",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "08-12 45 67",
```

Comment:
Are such short number still common?

Reply:
This specific format seems quite prone to false positives, especially with such high score (0.9).

Reply:
@tellarin - could you please elaborate a little around scoring and its meaning in this situation? Why is a score close to 1 not good?

Reply:
The more well formatted the number is and the more confident the recognizers are, the closer to 1 the score should be.
But for this particular case, in my opinion, it may be too prone to false positives, so this should ideally be somehow reflected in a lower score.
Are such short numbers still common? And specifically in this format?

Reply:
Thanks for the explanation. To be honest I don’t know how common this particular number construct is nowadays. I have simply followed the PTS standard for each possible construct a Swedish phone number can have. By false-positives, do you mean there’s a risk it will show a match for a non-swedish phone number? Just so I’ve got it right: this recognizer should match swedish phone numbers and swedish phone numbers only, right?

Reply:
By false-positives I mean, for example, matching something that is not a phone number. Maybe a date followed by some numbers. Or other cases.

Reply:
Do you have a link to the PTS standard? If so, can you add it in the PR description?

Reply:
@tellarin - I have added a reference to the standard in the description.

Reply:
@tellarin - I could extend the specs w/ an array of cases containing a mix of phone numbers and other numbers (like the english spec) to ensure the recognizer doesn’t hit these. Should I do that?

Reply:
@pete1854, that would be very helpful. Specially phrases with partial date and time, other numerical tokens, IDs, etc.
We don't need too many, but a couple cases focused on patterns that may be similar to one of the new phonenumber formats. Thanks!

Reply:
Based on the standard you linked, it seems the only NDC that can fall into the easier to mismatch format is Stockholm (08).
So let's at least add an extra validation that only numbers starting with 08 can following the "0N-AX XX XX" format.

Reply:
The others with 0xx-xxx or 0xx-xxx are much less likely to be problematic, I think.